sdk/websites/Microsoft.Azure.Management.WebSites/src/Generated/Models/Certificate.cs

```diff
@@ -236,7 +236,8 @@ public Certificate(string location, string id = default(string), string name = d
         public string CanonicalName { get; set; }
 
         /// <summary>
-        /// Gets or sets method of domain validation for free cert
+        /// Gets or sets method of domain validation for free cert. Possible values include:
```

Comment:
I don't see any updates to the metadata for weisites, did you change this file manually?